server/auth/store_test.go

```diff
@@ -155,6 +157,13 @@ func TestUserAdd(t *testing.T) {
 	if err != ErrUserEmpty {
 		t.Fatal(err)
 	}
+
+	as.rangePermCacheMu.RLock()
+	defer as.rangePermCacheMu.RUnlock()
```

Comment:
I don't understand why do we need to acquire the lock in unit test. Is there any potential race condition here? Of course, it isn't a big deal to keep it as it is. 

The same comment to some following test cases.